src/services/webPlatformUtils.service.ts

```diff
@@ -285,4 +285,12 @@ export class WebPlatformUtilsService implements PlatformUtilsService {
     supportsSecureStorage() {
         return false;
     }
+
```

Comment:
Should we include some comments or something that this is stubbed out until we want to implement it?

Reply:
not sure comments are really needed, this is filling in for the contract... when we get around to supporting themes in web it should be pretty straightforward. That's why `null` is a bit more explanatory IMO.